uwblueprint/community-fridge-kw
# feat: landing, login, and sign up welcome screens

## Notion ticket link

[Welcome Page](https://www.notion.so/uwblueprintexecs/Landing-Page-Welcome-Screen-ceaaf3f753e543b49556dda4cab90e92)



## Implementation description
* Created custom themes based off Figma
* Add in initial landing screen


https://user-images.githubusercontent.com/19617248/139786897-c17c18db-34a1-42a2-9c1f-17f9fa1d56be.mp4



## Steps to test
1. Verify everything renders properly and **please run locally**
<img width="200" alt="Screen Shot 2021-11-02 at 12 28 07 AM" src="https://user-images.githubusercontent.com/19617248/139787001-429e5cbb-c208-4a65-a4b8-a21ff7b4fba0.png">
<img width="200" alt="Screen Shot 2021-11-02 at 12 28 12 AM" src="https://user-images.githubusercontent.com/19617248/139787002-bac71ef4-593c-41d1-99b7-df285dc90360.png">
<img width="200" alt="Screen Shot 2021-11-02 at 12 28 16 AM" src="https://user-images.githubusercontent.com/19617248/139787003-3358e4dc-36a5-482a-83af-98cc2580ed8a.png">
<img width="200" alt="Screen Shot 2021-11-02 at 12 28 22 AM" src="https://user-images.githubusercontent.com/19617248/139787005-0dd3a736-d688-4224-9e3b-3e2b4633968b.png">
<img width="200" alt="Screen Shot 2021-11-02 at 12 28 28 AM" src="https://user-images.githubusercontent.com/19617248/139787006-2c310cfb-df93-439f-a919-c43ce3f193e7.png">
<img width="200" src="https://user-images.githubusercontent.com/19617248/140016334-e3244f97-2471-4102-bcd2-26ecc50448d2.png" />
<img width="200" src="https://user-images.githubusercontent.com/19617248/140016349-d5474bf2-3326-4c97-87a4-aaa67b28961d.png" />




## Checklist
